platforms/registry/src/index.ts

```
@@ -1,13 +1,62 @@
+        server.log.info("Database connection initialized");
+    } catch (error) {
+        server.log.error("Error during database initialization:", error);
+        process.exit(1);
+    }
+};
+
+// Initialize VaultService
+const vaultService = new VaultService(AppDataSource.getRepository("Vault"));
+
+// Middleware to check shared secret
+const checkSharedSecret = async (request: any, reply: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)